pkg/controller/v1alpha1/trainedmodel/controller_test.go

```diff
@@ -134,36 +132,9 @@ var _ = Describe("v1beta1 TrainedModel controller", func() {
 					Name:      serviceKey.Name,
 					Namespace: serviceKey.Namespace,
 				},
-				Spec: v1beta1.InferenceServiceSpec{
```

Comment:
why these are deleted?

Reply:
This test case is when inference service is not ready. I wanted to create an inference service that fails because its conditions fail. Hence why I did not include a spec.